Manifests/ManagedPreferencesApplications/com.1password.1password.plist

```diff
@@ -144,6 +161,148 @@
 			<key>pfm_type</key>
 			<string>string</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>8.0</string>
```

Comment:
Given that the entire manifest is supported on version 8.0+, I doubt it is necessary to re-mention it for each of the properties.

I know that existing properties already had it mentioned on them and that maybe you just used the existing convention, but since we're making changes to this manifest - would you mind removing these from everywhere except the root dictionary, if it's not too much work?